core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -129,10 +130,11 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
     line-height: 0.6;
     padding-left: 4px;
     text-overflow: clip;
+    text-align: center;
```

Comment:
Ah, I don't think that's desirable. I think @markhalpin intended that the left edge of AUTHOR PROFILES matches the left edge of the usernames.